.github/workflows/5_builderpackage_server.yml

```diff
@@ -117,7 +118,7 @@ jobs:
           if [ "${{ inputs.docker_image_tag }}" == "auto" ]; then echo "TAG=$VERSION" >> $GITHUB_ENV;
           elif [ "${{ inputs.docker_image_tag }}" == "developer" ]; then echo "TAG=$(sed 's|[/\]|--|g' <<< ${{ github.ref_name }})" >> $GITHUB_ENV;
           else echo "TAG=${{ inputs.docker_image_tag }}" >> $GITHUB_ENV; fi
-          echo "CONTAINER_NAME=pkg_${{ inputs.system }}_server_builder_amd64" >> $GITHUB_ENV
+          echo "CONTAINER_NAME=pkg_${{ inputs.system }}_server_builder_${{ inputs.architecture }}" >> $GITHUB_ENV
```

Comment:
This is more tricky than expected.

 There are two archs but each package manager naming them differently
- deb: amd64/arm64
- rpm: x86_64/aarch64

Now, in order to avoid creating 4 containers, they align the 4 names into deb ones. This should be considered here too

Ref https://github.com/wazuh/wazuh/blob/v4.11.2/.github/workflows/packages-build-manager.yml#L119-L128

Reply:
To be consistent with the package [generation](https://github.com/wazuh/wazuh-agent/blob/77b30de556fec114d2c82db9588ac2b81ca9ef1c/.github/workflows/5_builderpackage_agent-linux.yml#L15-L23) of the agent, I decided to keep the actual nomeclature.

Reply:
Fixed in b30295486721b2aa2b525f81f1b46a17d0120e82